src/main/java/com/ashkan/moviesapi/entities/ActorsInMovies.java

```diff
@@ -8,16 +8,18 @@
 @Entity
 @Table(name = "actors_in_movies", schema = "moviedb", catalog = "")
 public class ActorsInMovies {
+    @JsonIgnore
     private int actorsInMoviesId;
     private int actorId;
+    @JsonIgnore
     private int movieId;
     @JsonIgnore
     private Actor actorByActorId;
     @JsonIgnore
     private Movie movieByMovieId;
```

Comment:
This is more a suggestion, I think is not necessary to name this property **movieByMovieId**, this is an object **movie**, that belongs to this ActorsInMovies Entity, it is implicit that this movie is filtered by the movieId, so you can name it directly **movie**.

Reply:
done.